style.css

```diff
@@ -1,12 +1,17 @@
 @import url("https://fonts.googleapis.com/css?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
 
+* {
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
 body {
-  background-color: white;
+  background-color: #e5e5e5;
   font-family: "Poppins", sans-serif;
   margin: 0;
   padding: 0;
-  width: 327;
-  height: 146;
+  width: 327px;
+  height: 146px;
```

Comment:
[OPTIONAL]

Even though this is optional I highly recommend you do it before you move forward, it will help you in the next projects.

- [ ] Please know that you shouldn't fix the width, mobile screens don't have the same width and what you're designing is meant to be viewed on a mobile screen, if you remove the width it will be way better and it will be responsive for many mobile screens, you should try that it's only one line then you can merge. 😁